de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1193,9 +1193,70 @@ Now, in the _Navigator_ double click on ConfigurationTree and scroll to the Mass
 Here you can find a budget breakdown.
 |image:chapterModeling/MassConceptBudget.png[Insepcting the mass budegt breakdown]
 
+|===
+==== Functional Electrical Architecture
+===== Add Interface to the model
+Virtual Satellite enables the user to add _Interfaces_ to the system model. Add the concept *Functional Electrical Architecture* as described in section in <<Activating a Concept>>
+Elements of the functional electrical architecture can be added to the system model by opening the context menu with the right mouse and click. 
+
+There are _InterfaceEnds_ that are of a specific _InterfaceType_  and have the potential to connect _Interfaces_.
+
+_InterfaceTypes_  need to be defined as child elements of an _InterfaceTypeCollection_ .
+The _InterfaceTypeCollection_  is created by clicking with the right mouse on the system model and select menu:Context[Functional Electrical Architecture>Add InterfaceTypeCollection].

```

Comment:
Maybe "system model" -> "Repository" would be a bit more precise.

Reply:
Done